tests/onnx/test_onnx_model_export.py

```diff
@@ -383,12 +342,10 @@ def test_log_model_no_registered_model_name(onnx_model, onnx_custom_env):
         mlflow.register_model.assert_not_called()
 
 
-# TODO: Mark this as large once MLflow's Travis build supports the onnxruntime library
```

Comment:
We no longer use Travis & GitHub actions doesn't have any trouble with `onnxruntime` :)